examples/pytorch/correct_and_smooth/model.py

```diff
@@ -94,10 +99,23 @@ def forward(self, g, labels, mask=None, post_step=lambda y: y.clamp_(0., 1.)):
 
             for _ in range(self.num_layers):
                 # Assume the graphs to be undirected
-                g.ndata['h'] = y * norm
+                if self.adj != 'AD':
+                    # DAD or DA
+                    y = norm * y
+                    if self.adj == 'DA':
+                        y = norm * y
```

Comment:
Compute two types of normalizations above so that we do not need to multiply twice for DA and AD.